frontend/packages/topology/integration-tests/features/topology/export-application/export-of-applicaton.feature

```diff
@@ -0,0 +1,54 @@
+@odc-2148
+Feature: Export of application
+              As a user, I have an unmanaged application which I want to export. I'd like to be able to later add that code to git or some shared location so that I can share with others, or import into a new cluster or same cluster but different project, or be able to apply updates to an existing application.
+
+
+
+    #The operator mentioned below is not yet available on operator hub
+        Background:
+            Given user has installed Gitops primer Operator
+              And user has created or selected namespace "aut-export-application"
+
+
+        @smoke @to-do
+        Scenario: Export application button in topology: EA-02-TC01
+            Given user is at Developer perspective
+              And user has created a deployment workload
+             When user goes to Topology
+              And user clicks on Export Application button
+             Then user can see a toast message saying "Export of resources in aut-export-application has started."
+              And user can see primer job created in topology
+
+
+        @regression @to-do
+        Scenario: Export Application modal when export application is in progress: EA-02-TC02
+            Given user is at Topology
+             When user clicks on Export Application button
+              And user clicks on Export Application button again
+             Then user can see a Modal saying "Export Application in primer app is in progress" with Ok button
+
+
+        @regression @to-do
+        Scenario: Export application button in empty topology view: EA-02-TC03
+            Given user is at Developer perspective
```

Comment:
Remove this line and move it to BackGround section

Reply:
Updated